manifests/plugin/discovery.pp

```diff
@@ -33,15 +33,19 @@
   if $install_images {
     include ::tftp::params
 
-    foreman::remote_file {"${::tftp::params::root}boot/${kernel}":
-      remote_location => "${source}${kernel}",
+    foreman::remote_file {"${::tftp::params::root}boot/${image_name}":
```

Comment:
Thanks for the example, it makes our lifes *much* more easier :-)

Gonna try that now.